Java/src/com/couchbase/litecore/C4Replicator.java

```diff
@@ -172,6 +184,21 @@ private static void documentEndedCallback(long handle, boolean pushing,
         }
     }
 
+    private static boolean validationFunction(String docID, int flags, long dict, boolean isPush, long ctx) throws Exception {//
```

Comment:
I don't think the Exception is ever thrown from this method. If it's true, please remove throws Exception.